CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Cancer case HPO panel variants link
 - Fix so that some drop downs have correct size
 - First IGV button in str variants page
+- Cancer case activates on SNV variants
```

Comment:
What about SV variants? Because we have those as well now!

Reply:
Already working.

Reply:
Ah ok!